### DISCUSSION TITLE
post/how-to-work-with-legacy-library/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#43] [Announcements]
# post/how-to-work-with-legacy-library/

10년도 전에 만들어진 SOAP 통신의 Library를 현재 REST 개발 환경에서 연동하기까지 있었던 문제점들과 해결 방법을 소개합니다.

https://tech.kakaopay.com/post/how-to-work-with-legacy-library/

**Comment by pay-dan-dy** [comment]
레거시 시스템과 연동할 때 겪게 되는 소중한 경험 공유 감사합니다!
